aws_lambda_powertools/metrics/provider/base.py

```diff
@@ -179,8 +179,13 @@ def handler(event, context):
         e
             Propagate error received
         """
+        extra_args = {}
 
-        default_dimensions = kwargs.get("default_dimensions")
+        if kwargs.get("default_dimensions"):
+            extra_args.update({"default_dimensions": kwargs.get("default_dimensions")})
+
+        if kwargs.get("default_tags"):
+            extra_args.update({"default_tags": kwargs.get("default_tags")})
```

Comment:
you don't need it ;) `**kwargs` will expand to key=value in the function call chain, so `log_metrics(..., default_tags={}` if default tags was passed

Reply:
Done.